### ISSUE TITLE
Support instance modify endpoint

### ISSUE THREAD

**Issue by xi-yang** [sdn-sense/sense-o-py-client#28]
@POST
    @Path("/{siUUID}/modify")
    @Consumes("application/json")

**Comment by xi-yang** [comment]
Modify with new service profile using -f option supported. Released as 1.28.

We need to support a delegated profile with edit options for modify. 
https://github.com/esnet/StackV/issues/1183#issuecomment-1751745598

For example 

sense_util.py -mo  -f below-profile.json -u service_instance_uuid

```
{
  "service_profile_uuid": "b61de552-138c-4f91-b64b-c6bf17ce4d42",
  "queries": [
    {
      "ask": "edit",
      "options": [
        {
          "data.connections[0].bandwidth.capacity": "5000"
        }
      ]
    }
  ]
}
```

**Comment by xi-yang** [comment]
verified working with SENSE-O patched.
